runtime/doc/usr_05.txt

```diff
@@ -378,17 +372,102 @@ The generic names for the filetype plugins are: >
 Here "<name>" can be any name that you prefer.
 Examples for the "stuff" filetype on Unix: >
 
-	~/.vim/ftplugin/stuff.vim
-	~/.vim/ftplugin/stuff_def.vim
-	~/.vim/ftplugin/stuff/header.vim
+	~/.nvim/ftplugin/stuff.vim
+	~/.nvim/ftplugin/stuff_def.vim
+	~/.nvim/ftplugin/stuff/header.vim
 
 The <filetype> part is the name of the filetype the plugin is to be used for.
 Only files of this filetype will use the settings from the plugin.  The <name>
 part of the plugin file doesn't matter, you can use it to have several plugins
 for the same filetype.  Note that it must end in ".vim".
 
+USING PLUGIN BUNDLES                                      *plugin-bundles*
+
+The manual method described above has the problem that things can get messy
+once you install a few plugins, specially if they include more than one file
+(which is common).  A way to solve this is to install plugins in their own
+directories and then add the path of those to the 'runtimepath' setting (so
+Vim will use the files from those directories when looking for plugins).  In
+practice, this means that instead of having a |vimfiles| tree like >
+
+	~.nvim/
+	    plugin/
+		a.vim
+	    ftplugin/
+		b.vim
+		e.vim
+	    syntax/
+		b.vim
+
+you can have something like this >
+
+	~.nvim/
+	    bundles/
+		a/
+		    plugin/
+			a.vim
+		b/
+		    ftplugin/
+			b.vim
+		    syntax/
+			b.vim
+		e/
+		    ftplugin/
+			e.vim
+
+This method allows the plugins to be distributed as their own trees, so they
+can be installed without affecting other plugins.  This is very helpful when
+synchronizing with a version control repository.
+
+So supposing the user wants to install a plugin that's distributed as this tree: >
+
+	autoload/
+	    awesome.vim
+	plugin/
+	    awesome.vim
+	doc/
+	    awesome.txt
+
+from http://github.com/developer/awesome-vim, they can do so by (assuming there
+is no "bundles" directory)
+
+    1. creating the plugin directory >
+
+	$ mkdir -p ~/.nvim/bundles/awesome-vim
+
+<    2. copying the files into the "bundles/awesome-vim/" folder >
+
+	$ cp awesome-vim/* ~/.nvim/bundles/awesome-vim/
+
+<    3. adding >
+
+	set runtimepath+=~/.nvim/bundles/awesome-vim
+
+<       to their |vimrc|.
+
+It is very likely that steps 1-2 can be replaced with something like: >
+
+    $ mkdir -p ~/.nvim/bundles/
+    $ cd ~/.nvim/bundles
+    $ git clone https://github.com/developer/awesome-vim
+
+PLUGIN MANAGERS                                            *plugin-managers*
+
+Bundle-style plugins can be handled by so-called "plugin managers", which are
+special plugins whose purpose is to handle 'runtimepath' and help the user to
+install, update and load plugins.
+
+The most popular plugin managers are:
```

Comment:
Instead of "The most popular", for reasons of maintenance and future-proofing, maybe we should say "Some great plugin managers are" or "Recommended plugin managers are" or something like that.